tasks/agent.py

```diff
@@ -173,8 +173,11 @@ def refresh_assets(ctx, build_tags, development=True, puppy=False):
         # copy the dd-agent placeholder to the bin folder
         bin_ddagent = os.path.join(BIN_PATH, "dd-agent")
         shutil.move(os.path.join(dist_folder, "dd-agent"), bin_ddagent)
+
+    # Network tracer not supported on windows
+    if sys.platform != 'win32':
```

Comment:
Is there a better way to do this? It won't be generated for android or windows

Reply:
right now this is the best way to do this 👍 

This would copy the config on macOS as well, is it intended? (if it isn't, let's only copy on Linux, i.e. `sys.platform.startswith('linux')`)